spacepy/pybats/__init__.py

```diff
@@ -1714,7 +1714,7 @@ class ImfInput(PbData):
     '''
     A class to read, write, manipulate, and visualize solar wind upstream
     input files for SWMF simulations.  More about such files can be found
-    in the SWMF/BATS-R-US documentation for the \#SOLARWINDFILE command.
+    in the SWMF/BATS-R-US documentation for the \\#SOLARWINDFILE command.
```

Comment:
Quick question.
In the original, the single backslash in `\#SOLARWINDFILE` was to escape `#` (I think.) Now, we're escaping `\`, which I don't think is necessary. I think we just want to remove the `\\` preceding the hash, unless I'm missing something with Sphinx...